Docs: Document jaeger tracing instrumentation (#21392)
grafana/grafana: docs/sources/administration/jaeger-instrumentation.md

+++
title = "Jaeger instrumentation"
description = "Jaeger traces emitted and propagation by Grafana"
keywords = ["grafana", "jaeger", "tracing"]
type = "docs"
[menu.docs]
parent = "admin"
weight = 9
+++

> # Jaeger instrumentation

Reviewer: When we get a better handle on the final shape of the documentation, we should revisit this title. I suspect we can come up with something more descriptive and accurate.

> Grafana supports [jaeger tracing](https://www.jaegertracing.io/).

Reviewer: ```suggestion
Grafana supports [Jaeger tracing](https://www.jaegertracing.io/).
```

> Grafana can emit jaeger traces for its http api endpoints as well as propagate jaeger trace information to datasources. 

Reviewer: ```suggestion
Grafana can emit Jaeger traces for its HTTP API endpoints and propagate Jaeger trace information to data sources. 
```

Reviewer: Unclear if "its HTTP API endpoints" are Grafana's or Jaeger's. Please clarify.

> All HTTP endpoints are logged evenly. (annotations, dashboard, tags, etc)

Reviewer: ```suggestion
All HTTP endpoints are logged evenly (annotations, dashboard, tags, and so on).
```

When a trace ID is propagated, it is reported with operation 'HTTP /datasources/proxy/:id/*'.

> Jaeger tracing is enabled in the [tracing.jaeger] section in your [grafana.ini](https://grafana.com/docs/grafana/latest/installation/configuration/#tracing-jaeger) config file.

Reviewer: ```suggestion
Refer to [Configuration](https://grafana.com/docs/grafana/latest/installation/configuration/#tracing-jaeger) for information about enabling Jaeger tracing.
```
